x/group/spec/group.md

```diff
@@ -46,75 +29,28 @@ type Tally struct {
 
 ```go
 type MsgCreateGroup struct {
-	// The Owner of the group is allowed to change the group structure. A group account
+	// The admin of the group is allowed to change the group structure. A group account
 	// can own a group in order for the group to be able to manage its own members
-	Owner  sdk.AccAddress `json:"owner"`
+	Admin  sdk.AccAddress `json:"admin"`
 	// The members of the group and their associated power
 	Members []Member `json:"members,omitempty"`
-	Description string `json:"Description,omitempty"`
-    DecisionProtocol DecisionProtocol `json:"decision_protocol"`
+	Description string `json:"description,omitempty"`
 }
 ```
 
-*Returns:* `sdk.AccAddress` based on an auto-incrementing `uint64`.
-
-### `MsgGroupExec`
-
-```go
-// MsgGroupExec executed the provided messages using the groups account if the
-// provided signers pass the group's DecisionProtocol. This is essentially a
-// basic multi-signature execution method.
-type MsgGroupExec struct {
-    Group sdk.AccAddress `json:"group"`
-    Signers []sdk.AccAddress `json:"signers"`
-    Msgs []sdk.Msg `json:"msgs"`
-    // Policy specifies the policy by which this propose will pass or fail.
-    // If set to DefaultGroupPolicy, the group's root DecisionProtocol is used. If
-    // it is set to another GroupPolicyID for this group, that policy's
-    // DecisionProtocol and its Capability grants will be used.
-	Policy   GroupPolicyID `json:"policy"`
-}
-```
+*Returns:* `GroupID` based on an auto-incrementing `uint64`.
 
-### `MsgUpdateGroupMembers`
+### `MsgUpdateGroup
 
 ```go
 // MsgUpdateGroupMembers updates the members of the group, adding, removing,
 // and updating members as needed. To remove an existing member set its Power to 0.
 type MsgUpdateGroupMembers struct {
-	Owner  sdk.AccAddress `json:"owner"`
-	Group  sdk.AccAddress `json:"group"`
-	Members []Member `json:"members,omitempty"`
-}
-```
-
-### `MsgUpdateGroupOwner`
-
-```go
-type MsgUpdateGroupOwner struct {
-	Owner  sdk.AccAddress `json:"owner"`
-	Group  sdk.AccAddress `json:"group"`
-	NewOwner  sdk.AccAddress `json:"new_owner"`
-}
-```
-
-### `MsgUpdateGroupDecisionProtocol`
-
-```go
-type MsgUpdateGroupDecisionProtocol struct {
-	Owner  sdk.AccAddress `json:"owner"`
-	Group  sdk.AccAddress `json:"group"`
-    DecisionProtocol DecisionProtocol `json:"decision_protocol"`
-}
-```
-
-### `MsgUpdateGroupDescription`
-
-```go
-type MsgUpdateGroupDescription struct {
-	Owner  sdk.AccAddress `json:"owner"`
-	Group  sdk.AccAddress `json:"group"`
-    DecisionProtocol DecisionProtocol `json:"decision_protocol"`
+	Admin  sdk.AccAddress `json:"admin"`
+	Group  GroupID `json:"group"`
+	NewAdmin  sdk.AccAddress `json:"new_admin"`
```

Comment:
We can either make it a no-op like you're proposing or this would seal the group and make it impossible to change? Is that desirable? I think that would probably be unexpected behavior so for now I will define it as a no-op. If there is a use case for that, this could be a separate "seal group" message.

Reply:
I agree